llvm/lib/Transforms/Utils/CallPromotionUtils.cpp

```diff
@@ -185,12 +187,30 @@ static void createRetBitCast(CallBase &CB, Type *RetTy, CastInst **RetBitCast) {
     U->replaceUsesOfWith(&CB, Cast);
 }
 
+// Returns the or result of all icmp instructions.
+static Value *getOrResult(const SmallVector<Value *, 2> &ICmps,
```

Comment:
Nit: change the name to getORResult or just getORs

Reply:
done.